Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -1015,14 +2152,18 @@ def import_ioc_without_approval(client: Client, file_id, classification, confide
     return "The data was imported successfully."
 
 
-def get_model_list(client: Client, model, limit="50"):
+def get_model_list(client: Client, model, limit="50", page=None, page_size=None):
     """
         Returns list of Threat Model that was specified. By default limit is set to 50 results.
         Possible values for model are : actor, campaign, incident, signature, ttp, vulnerability, tipreport
     """
     # if limit=0 don't put to context
-    params = dict(limit=limit, skip_intelligence="true", skip_associations="true")
-    model_list = client.http_request("GET", F"v1/{model}/", params=params).get('objects', None)
+    params = return_params_of_pagination_or_limit(arg_to_number(page), arg_to_number(page_size), arg_to_number(limit))
+    params.update(dict(skip_intelligence="true", skip_associations="true", order_by='-created_ts'))
+    url = F"v1/{model}/"
+    if model == 'attack pattern':
+        url = F"v1/{model.replace(' ', '')}/"
```

Comment:
```suggestion
        url = f"v1/{model.replace(' ', '')}/"
```

Reply:
Just put in the attack pattern in the suffix as it should be, no need to use replace